Initial Portland 2022 website (#1637)
writethedocs/www: docs/conf/portland/2022/news/welcome.rst

:template: {{year}}/generic.html

.. post:: Dec 15, 2022
   :tags: {{shortcode}}-{{year}}, website, tickets

Announcing Write the Docs {{ city }} {{year}}
===========================================

Welcome to the official launch of `Write the Docs Portland {{year}} <https://www.writethedocs.org/conf/portland/{{year}}/>`_.
We're excited to announce the dates for our next conference:
**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**

> We are planning to do a **hybrid event** this year,
> which means the in-person part of the conference will be returning.
> We are still in the planning phases for how to handle this safely,
> and would love your `feedback <mailto:portland@writethedocs.org>`_ on how we should progress.

Reviewer: ```suggestion
We will be running a **hybrid event** this year, our online event joining a return to the in-person part of the conference.
We are still in the planning phase for how to handle this safely,
and would love your `feedback <mailto:portland@writethedocs.org>`_ on how we should progress.
```

.. TODO: Add link to survey?

It's been a rough year for a lot of us,

> but we're hoping that late next spring things will be looking a lot safer for us to gather.

Reviewer 2: ```suggestion
but we're hoping that by next spring things will be looking a lot safer for us to gather.
```

> It's been a rough year for a lot of us,
> but we're hoping that late next spring things will be looking a lot safer for us to gather.
> We plan to release more details during the next few weeks,
> but there's a few specific updates below.

Reviewer: ```suggestion
It's been a rough year for a lot of us,
but we're hoping that by the third week of May things will be looking a lot safer for us to gather in-person.
We will be releasing more details at the start of 2022,
but there are a few specific updates below.
```

Health Guidelines
-----------------

We will continue to monitor the health guidelines for Portland & Oregon as the conference approaches,
but our current plan is to have the following rules in place:

* Vaccination required for all attendees

> * Masks required for all indoor spaces

Reviewer 2: ```suggestion
* Masks required in all indoor spaces
```

* A new venue that provides more access to outside air, and outdoor space
* 100% refund for all attendees who need to cancel for COVID-related reasons
* Limited venue capacity to give people enough room to socially distance

> * Seating options based on comfort level being near others in various spaces

Reviewer 2: ```suggestion
* Seating options based on distancing comfort level in various spaces
```

> * Venue spaces with windows that open or that are fully outside

Reviewer: Something something ventlation instead of talking about windows?

Reviewer 3: Yup, it sounds like some windows are fully outside... ;) 

```suggestion
* Indoor meeting spaces with open windows (weather permitting)
* Outdoor meeting areas
```

> We also have a few more ideas to ensure safety of all attendees that we are discussing,
> and will have more information as the event comes closer.

Reviewer: ```suggestion
We are discussing more ideas and processes to ensure safety of all attendees, and will publish more information as we get closer to the event.
```

Call for Proposals
------------------

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!

> We plan to launch our CFP in January and have it last until sometime in February.

Reviewer: ```suggestion
We plan to launch our CFP in January for those interested in speaking.
```

Reviewer 2: A couple of months sounds like a long time, are we still planning to leave 3 months between announcing speakers and the conference? Don't forget we need to give them time to record their talks...

Author: Yea, I'll just not be specific.

We're still working out the exact details for the CFP,

> but our current plan is to record all talks so that the experience will be comparable between online and in-person attendees.

Reviewer: ```suggestion
but our current plan is to record all talks so that the experience will be comparable between online and in-person, for attendees and speakers.
```

> We hope this will reduce anxiety for our speakers about feeling the need to attend in person,
> or have a second-rate experience presenting remotely.

Reviewer 2: ```suggestion
We hope that this will help maintain the accessibility level that remote presentations gave us and invite speakers who cannot or do not feel comfortable attending in person.
```

Reviewer 2: I'd rather focus on accessibility than anxiety ;)

> Get started coming up with ideas for the conference now,
> and we'll let you know when the CFP opens.

Reviewer: ```suggestion
Start thinking about your talk proposal now, and we're let you know as soon as the CFP opens.

<!-- insert mailing list box? -->
```

Tickets
-------

Tickets will go on sale in **{{ date.tickets_live }}**.
We will announce ticket prices soon,
so that you can start planning your budget.
As with the previous years, we have tiered pricing to make it fair for everyone,

> and will likely price in-person and virtual tickets separately.

Reviewer: ```suggestion
and will price in-person and virtual tickets separately.
```

As always, we keep ticket prices low so that it is accessible to the widest range of people possible.
However, if you can't afford these prices and still wish to attend,
we will be running our Opportunity Grants program again this year.

Sponsorship
-----------

We are still thinking about how best to handle sponsorship for a hybrid event,
and will release our prospectus soon.

> If you are already excited about sponsoring,
> you can always reach out to us now.
> We're always open to customization on our sponsorships, especially given the new format.
> You can always reach us with ideas or questions at sponsorship@writethedocs.org.

Reviewer: ```suggestion
If you are already excited about sponsoring, reach out to us at sponsorship@writethedocs.org.
We're open to customization or new ideas for our sponsorships levels, especially given the new format.
```

> Stay Updated

Reviewer: ```suggestion
Keep in touch
```

------------

Want to find out what's happening with the conference, or enjoy our monthly global community newsletter?
Sign up to one or more of our `mailing lists <http://eepurl.com/cdWqc5>`_. Your information will never be shared with any third-parties, and you can unsubscribe at any time.

Want to connect with other documentarians in real-time? Join our `Slack <https://www.writethedocs.org/slack/>`_.

Conclusion
----------

> We hope you will join us in Portland for another Write the Docs conference.

Reviewer: ```suggestion
We hope you will join us in Portland, or online, for another Write the Docs conference.
```
